tests/unit/Codeception/Module/FacebookTest.php

```diff
@@ -117,21 +116,9 @@ public function testLoginToFacebook()
         // preconditions: #3 test user logged in on facebook
         $this->module->haveTestUserLoggedInOnFacebook();
 
-        // go to our page with facebook login button
-        $browserModule->amOnPage('/facebook');
```

Comment:
Thanks for the feedback, I have added the changes. About the facebook login test, it did not work for me and it was marked with markTestSkipped(). I did try to fix it but I see now that I misunderstood it a little bit. I try to see if I can fix this first thing tomorrow morrning.
Also for seePostOnFacebookWithAttachedPlace the new API did not return placeId, but I will check if it is possible to do this in some other way so that seePostOnFacebookWithAttachedPlace($placeId) is not removed.